<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Soy el label 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUno">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Soy el LineEdit 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUno">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar informaci√≥n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbrir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir ventana 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
